### PR TITLE
Create mini-driver/config setup for SplitExplicit tests.

### PR DESCRIPTION
**Pull request by blallen**
# Description

Splitting off some quality of life improvements from #1482 to make that PR easier to follow. This PR splits the time stepper setup off from the model setup, so we can run different scenarios with the split-explicit solver.

I have

- [x] Written and run all necessary tests with CLIMA by including `tests/runtests.jl`
- [x] Followed all necessary [style guidelines](https://clima.github.io/ClimateMachine.jl/latest/DevDocs/CodeStyle/) and run `julia .dev/climaformat.jl .`
- [ ] Updated the documentation to reflect changes from this PR.

# For review by CLIMA developers

- [x] There are no open pull requests for this already
- [x] CLIMA developers with relevant expertise have been assigned to review this submission
- [x] The code conforms to the [style guidelines](https://clima.github.io/ClimateMachine.jl/latest/DevDocs/CodeStyle/) and has consistent naming conventions. `julia .dev/format.jl` has been run in a separate commit.
- [x] This code does what it is technically intended to do (all numerics make sense physically and/or computationally)

### CODE REVIEW

**Comment by blallen** [comment]
bors try

**Comment by blallen** [comment]
bors try

**Comment by blallen** [comment]
bors try

File: test/Ocean/SplitExplicit/split_explicit.jl [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+            CentralNumericalFluxFirstOrder(),
+            CentralNumericalFluxSecondOrder(),
+            CentralNumericalFluxGradient(),
+        )
+
+        Q_3D = init_ode_state(dg_3D, FT(0); init_on_cpu = true)
+        Q_2D = init_ode_state(dg_2D, FT(0); init_on_cpu = true)
+
+        lsrk_3D = LSRK54CarpenterKennedy(dg_3D, Q_3D, dt = dt_slow, t0 = 0)
+        # lsrk_3D = LS3NRK33Heuns(dg_3D, Q_3D, dt = dt_slow, t0 = 0)
```
Comment (thomasgibson) [inline test/Ocean/SplitExplicit/split_explicit.jl:152]: Comments like this can be removed

Comment (blallen) [reply test/Ocean/SplitExplicit/split_explicit.jl:152]: good catch

**Pull request merged**